eng/Baseline.xml

```diff
@@ -71,7 +71,6 @@ Update this list when preparing for a new patch.
   <Package Id="Microsoft.AspNetCore.Owin" Version="5.0.6" />
   <Package Id="Microsoft.AspNetCore.Razor.Language" Version="5.0.6" />
   <Package Id="Microsoft.AspNetCore.Server.Kestrel.Transport.Experimental.Quic" Version="5.0.6" />
-  <Package Id="Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv" Version="5.0.6" />
```

Comment:
I think this should stay because we did ship the package in 5.0. Won't matter much in a week when we update the baseline to 6.0 anyways, though.

Reply:
Yes, @Tratcher please leave this file and Baseline.Designer.props alone. I'll change the files when updating them to reflect 6.0.0 GA (soon 😺). For now, they should reflect what we shipped in 5.0.x.